tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -161,12 +161,45 @@ def on_heartbeat(self, cluster_data):
             if data:
                 self.on_sync_object(data)
 
+        # Sync the utilization details for the cluster
+        self._sync_utilization()
+
         # Get and update rbds for pools
         self._sync_rbds()
 
         # Get and update ec profiles for the cluster
         self._sync_ec_profiles()
 
+    def _sync_utilization(self):
+        util_data = self._get_utilization_data()
+        NS.ceph.objects.Utilization(
+            total=util_data['cluster']['total'],
+            used=util_data['cluster']['used'],
+            available=util_data['cluster']['available'],
+            pcnt_used=util_data['cluster']['pcnt_used']
+        ).save()
+
+        # Loop through the pools and update the utilization details
+        try:
+            pools = NS._int.client.read(
+                "clusters/%s/Pools" % NS.tendrl_context.integration_id,
+            )
+        except etcd.EtcdKeyNotFound:
+            # No pools so no need to continue with pool utilization sync
+            return
+
+        for entry in pools.leaves:
+            fetched_pool = NS.ceph.objects.Pool(
+                pool_id=entry.key.split("Pools/")[-1]
+            ).load()
+            pool_util_data = util_data['pools'].get(
+                fetched_pool.pool_name,
+                {}
+            )
+            fetched_pool.used = pool_util_data.get('used', 0)
```

Comment:
you should do 

fetched_pool.used = pool_util_data.get('used', fetched_pool.used)

same for pcnt_used